### PR TITLE
[FLINK-22562][docs-zh]Translate recent updates to backpressure and ch…

### PR DESCRIPTION
**Pull request by 95chenjz**
…eckpointing docs

## What is the purpose of the change

Translate recent updates to backpressure and checkpointing docs

## Brief change log

Update 'flink/docs/content.zh/docs/ops/monitoring/back_pressure.md'

## Verifying this change

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): no
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`:  no
  - The serializers:  no
  - The runtime per-record code paths (performance sensitive):  no
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn/Mesos, ZooKeeper:  no
  - The S3 file system connector:  no

## Documentation

  - Does this pull request introduce a new feature? no
  - If yes, how is the feature documented? not applicable

### CODE REVIEW

**Comment by 95chenjz** [comment]
cc @wuchong @pnowojski. Could you review my PR when you're free? Looking for your feedback. Thanks.

**Comment by 95chenjz** [comment]
cc @wuchong @xccui @klion26, looking for your feedbacks, thanks!

File: docs/content.zh/docs/ops/monitoring/back_pressure.md [hunk 1]
```diff
@@ -28,62 +28,51 @@ under the License.
...
+WebUI 集合了所有 subTasks 的反压和繁忙指标的最大值，并在 JobGraph 中将集合的值进行显示。除了显示原始的数值，tasks 也用颜色进行了标记，使检查更加容易。
 
-你可以在 Job 的 Overview 选项卡后面找到 *Back Pressure* 。
-
-### 采样进行中
+{{< img src="/fig/back_pressure_job_graph.png" class="img-responsive" >}}
 
-这意味着 JobManager 对正在运行的 Task 触发了反压采样。默认情况下，大约需要 5 秒完成采样。
+闲置的 tasks 为蓝色，完全被反压的 tasks 为黑色，完全繁忙的 tasks 被标记为红色。
+中间的所有值都表示为这三种颜色之间的颜色。
```
Comment (movesan) [inline docs/content.zh/docs/ops/monitoring/back_pressure.md:62]: 中间的所有值都表示为这三种颜色之间的"过渡色"。

File: docs/content.zh/docs/ops/monitoring/back_pressure.md [hunk 2]
```diff
@@ -28,62 +28,51 @@ under the License.
...
-
-你可以使用以下键来配置 JobManager 的样本数：
-
-- `web.backpressure.refresh-interval`: 有效的反压结果被废弃并重新进行采样的时间 (默认: 60000, 1 min)。
-- `web.backpressure.num-samples`: 用于确定反压采样的样本数 (默认: 100)。
-- `web.backpressure.delay-between-samples`: 用于确定反压采样的间隔时间 (默认: 50, 50 ms)。
+这些指标每两秒更新一次，上报的值表示 subtask 在最近两秒被反压（或闲或忙）的平均时长。
+当你的工作负荷是变化的时需要尤其引起注意。比如，一个以恒定50%负载工作的 subtask 和另一个每秒钟在满负载和闲置切换的 subtask 的`busyTimeMsPerSecond`值相同，都是`500ms`。 
 
+在内部，被压时根据输出 buffers 的可用性来进行判断的。
```
[the remark this answers is not in the document]
Comment (95chenjz) [reply docs/content.zh/docs/ops/monitoring/back_pressure.md]: "在内部，反压根据输出 buffers 的可用性来进行判断的。" could be a better explanation.

File: docs/content.zh/docs/ops/monitoring/back_pressure.md [hunk 3]
```diff
@@ -28,62 +28,51 @@ under the License.
...
 在工作流中数据记录是从上游向下游流动的（例如：从 Source 到 Sink）。反压沿着相反的方向传播，沿着数据流向上游传播。
 
-以一个简单的 `Source -> Sink` Job 为例。如果看到 `Source` 发生了警告，意味着 `Sink` 消费数据的速率比 `Source` 生产数据的速率要慢。
+以一个简单的 `Source -> Sink` job 为例。如果看到 `Source` 发生了警告，意味着 `Sink` 消费数据的速率比 `Source` 生产数据的速率要慢。
 `Sink` 正在向上游的 `Source` 算子产生反压。
 
+## Task 性能指标
 
-## 反压采样
+Task（SubTask）的每个平行实例都可以用三个一组的指标评价：
```
[the remark this answers is not in the document]
Comment (95chenjz) [reply docs/content.zh/docs/ops/monitoring/back_pressure.md]: Accepted.

[the remark this answers is not in the document]
Comment (95chenjz) [reply docs/content.zh/docs/ops/monitoring/back_pressure.md]: Accepted.

File: docs/content.zh/docs/ops/monitoring/back_pressure.md (hunk 1 above)
Comment (95chenjz) [reply docs/content.zh/docs/ops/monitoring/back_pressure.md]: Accepted.

Comment (95chenjz) [reply docs/content.zh/docs/ops/monitoring/back_pressure.md]: Thank you so much for your feedbacks, Movesan.

**Comment by pnowojski** [comment]
Hey, can this be merged?

**Comment by 95chenjz** [comment]
@PatrickRen Hi Patrick, could you give some feedbacks for my translation? Thank you so much!

**Comment by 95chenjz** [comment]
> Hey, can this be merged?

Yes, I think it's ready.

**Comment by 95chenjz** [comment]
Hi @wuchong, this pr is ready for merge. could you?

**Review by wuchong — APPROVED** [review]
LGTM.

**Pull request merged**